app/src/renderer/components/common/PagePreferences.vue

```diff
@@ -40,34 +36,47 @@
 <script>
 import { mapGetters } from "vuex"
 import TmListItem from "common/TmListItem"
-import TmBtn from "common/TmBtn"
 import TmPage from "common/TmPage"
 import TmPart from "common/TmPart"
 import TmField from "common/TmField"
+import TmFormMsg from "common/TmFormMsg"
 
 export default {
   name: `page-preferences`,
   components: {
-    TmBtn,
     TmField,
+    TmFormMsg,
     TmListItem,
     TmPage,
     TmPart
   },
+  data: () => ({
+    showError: false
+  }),
   computed: {
-    ...mapGetters([`session`, `onboarding`, `nodeUrl`])
+    ...mapGetters([`session`, `nodeUrl`])
   },
   methods: {
-    setErrorCollection({ $store } = this) {
-      $store.dispatch(`setErrorCollection`, {
-        address: this.session.address,
-        optin: !this.session.errorCollection
-      })
-    },
-    setOnboarding({ $store } = this) {
-      $store.commit(`setOnboardingState`, 0)
-      $store.commit(`setOnboardingActive`, true)
+    setErrorCollection() {
+      if (this.session.experimentalMode) {
+        this.showError = true
+        setTimeout(() => {
+          this.showError = false
+        }, 5000)
+      } else {
+        this.$store.dispatch(`setErrorCollection`, {
+          address: this.session.address,
+          optin: !this.session.errorCollection
+        })
+      }
     }
   }
 }
 </script>
+<style scoped>
+/* TODO: fix style */
+.tm-form-msg {
```

Comment:
@jbibla could you help?

Reply:
yes, this page is ugly. lol. let's merge and i'll fix.

Reply:
sure thing!

Reply:
honestly, i think we should just delete the preferences page at this point and remove it from the toolbar.

it only has node IP which we show in the network-connectivity toolbar and this switch which doesn't even work properly. 

this switch / toggle will appear in the cookie banner. but for now we don't need this page at all.

Reply:
then kill it 🔥

Reply:
killed